CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
   * Removed DynamicNavObstacle - an unused class (#732)
   * Merged networked data classes into their Server counterparts. An example of that change is the contents of NetworkCharacterState getting moved into ServerCharacter, contents of NetworkDoorState getting moved into SwitchedDoor etc. (#732)
 * Engine version bump to 2021.3.10f1 (#740)
+* VandalImp's ImpTossedItem attack only damages PCs & damage raised (#743)
```

Comment:
Since vandal imp is new for this release, should there be a line for this?
```suggestion
```

Reply:
That's correct! Reverted.